cmd/kev/cmd/ux.go

```diff
@@ -36,6 +36,14 @@ func displayCmdStarted(cmdName string) {
 	_, _ = os.Stdout.Write([]byte("> " + cmdName + "...\n"))
 }
 
+func displayDevModeStarted(envs []string) {
+	_, _ = fmt.Fprintf(os.Stdout, "\033[2m[development mode] ... listing environments:\n")
```

Comment:
`watched environments`?

Reply:
sure ... I borrowed the `skaffold` lingo ... `watched` make sense.